.gitignore

```diff
@@ -1,3 +1,5 @@
 _site
 .sass-cache
 .live-archive/
+Gemfile.lock
```

Comment:
NO! BAD!

This should only change when the dependencies change, which is a code change and should be in source control. Keeping this file in the repo eliminates a huge class of version mismatch bugs.

NPM (pre-5) is teaching bad habits.